src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt

```diff
@@ -183,4 +184,36 @@ class EdmTest : MultipleAuthenticatedUsersBase() {
                 "Entity type of id ${et.id} does not exists."
         )
     }
+
+    @Test
+    fun testEntityTypePropertyTypeMetadata() {
+        val pt1 = createPropertyType()
+        val et = createEntityType(pt1.id)
+        val update =  MetadataUpdate(
+                Optional.of("New Title !"),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty()
+        )
+        edmApi.updateEntityTypePropertyMetadata( et.id , pt1.id, update )
+        var metadata = edmApi.getEntityTypePropertyMetadata(et.id, pt1.id);
+        Assert.assertEquals(
+                "New Title !",
```

Comment:
This is nit-picky, but I'd probably make this a private val rather than a raw string